Multisig/UI/UI Library/Address/SlicedText+Address.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  SlicedText+Address.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 15.07.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+extension SlicedText.Style {
+    static let addressShortLight = Self.init(color: .gnoMediumGrey, truncate: .middle)
+    static let addressShortDark = Self.init(color: .gnoDarkBlue, truncate: .middle)
+    static let addressLong = Self.init(middleColor: .gnoMediumGrey, sideColor: .gnoDarkBlue, truncate: .none)
```

Comment:
@MouazAlzahabi now the styling is customizable - just add or remove or change in one place.

Reply:
really nice :)